lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -107,7 +114,7 @@ class TraceOfTab extends ComputedArtifact {
       firstPaint,
       firstContentfulPaint,
       firstMeaningfulPaint,
-      traceEnd,
+      traceEnd: latestTraceEvt,
```

Comment:
yeah what do you think about using navStart and traceEnd timings instead of all this other nonsense entirely?